blackbox/docs/configuration.txt

```diff
@@ -1899,6 +1912,80 @@ Javascript Language
   Setting to enable the Javascript language. As The Javascript language is an
   experimental feature and is not securely sandboxed its disabled by default.
 
+Authentication
+==============
+
+Authentication settings are node settings, which means that their values apply
```

Comment:
I would see it like `psql.enabled` and `psql.port`, where both are node settings